### DISCUSSION TITLE
WeakEvents and iOS

### DISCUSSION THREAD

**Discussion by Flying--Dutchman** [TheCodeTraveler/AsyncAwaitBestPractices#184] [Q&A] [answered]
Unfortunately, another question about WeakEvents under MAUI. 
I noticed that "System.Reflection.Emit" is used for WeakEvents. Does this pose a problem for iOS devices?
Or would that only be a problem if IL code was actually generated?

**Comment by TheCodeTraveler** [comment, marked as answer]
Nope, there's no concerns for iOS apps. 

I have a few apps in the App Store using this library with no issues.

**Reply by Flying--Dutchman** [reply]
Thank you
